mne/surface.py

```diff
@@ -229,6 +256,46 @@ def get_meg_helmet_surf(info, trans=None, verbose=None):
     return surf
 
 
+def _scale_helmet_to_sensors(system, surf, info):
+    fname = _helmet_path / f"{system}_ch_pos.txt"
+    if not fname.is_file():
+        return surf
+    with open(fname) as fid:
+        ch_pos_from = json.load(fid)
+    # find correspondence
+    fro, to = list(), list()
+    for key, f_ in ch_pos_from.items():
+        t_ = [ch["loc"][:3] for ch in info["chs"] if ch["ch_name"].startswith(key)]
+        if not len(t_):
+            continue
+        fro.append(f_)
+        to.append(np.mean(t_, axis=0))
+    if len(fro) < 4:
+        logger.info("Using CAD helmet because fewer than 4 sensors found")
```

Comment:
here (and the similar info message below) I wonder if it could be clarified / made more accurate.  IIUC, in either case you're always using the CAD helmet, the question is whether or not you affine xform the CAD model to match digitized sensor locs from `info`?  If that's correct, then to me a better message is something like

```suggestion
        logger.info(
            "Using helmet surface coords from file, because not enough sensors found "
            f"to support transforming to device coord frame (got {len(fro)}, need ≥4)."
        )
```